src/test/java/org/symphonyoss/symphony/messageml/elements/form/DatePickerTest.java

```diff
@@ -269,6 +269,86 @@ public void sendUnsupportedDateFormat() throws Exception {
     context.parseMessageML(input,null, MessageML.MESSAGEML_VERSION);
   }
 
+  @Test
```

Comment:
the tests are clear enough to me like that but they could also be implemented using parameterized tests as it seems we only play on the format value

Reply:
you are right, but maybe the next time we could think to move to JUnit5 (and parameterized are better on it)